### ISSUE TITLE
Review the updates of github terms of service

### ISSUE THREAD

**Issue by danielegobbetti** [Freeyourgadget/Gadgetbridge#725]
This is not a technical topic.

Github is updating its terms of service effective August 7th.
The updates are documented in a pull request of this repository: https://github.com/github/site-policy

I think it's wise to review them to identify possibly problematic changes for this project. Amendments could be submitted to the linked repo until July 28th.

**Comment by IzzySoft** [comment]
> Github is updating its terms of service effective August 7th.

And despite of them using a VCS for that code, we cannot diff as everything is just the "initial check-in". Not sure if I should be amused. That way it's not easy to "identify possibly problematic changes" as we cannot see what will be *changing.*

I eagerly followed their suggestion to add a "code of conduct"; but the link from the community guidelines document got a 404. Same for the guideline concerning contribution. As all those documents have been added a month ago I doubt I'm the first to notice; strange those links have not been fixed yet.

Other short notes: We can skip some documents as they don't apply to us.

* Ammendment to the TOS: only relevant for US based federal projects.
* Corporate TOS: we're no corporation (applies to business only)
* 2x Marketplace: Unless we plan to use that, it's not relevant to us. (we cannot "click the button below" anyhow as it's not there – looks like they simply copy-pasted content from another location and forgot to review)
* open-source-applications: Does not apply to the project, but to the use of specific Github software ("GitHub Desktop™")
* pre-release-program: specific to participation in GH pre-release testing (ie ***their*** pre-releases)
* registered developer: that's for a special GH programm one needs to explicitly sign-up for, so most likely not relevant to the project as such

**TL;DR** Basically, the important documents should be:

* [dmca-takedown-policy.md](https://github.com/github/site-policy/blob/master/Policies/dmca-takedown-policy.md) (this and the other two DMCA documents, just in case – which hopefully never ever affect us in the future)
* [github-community-guidelines.md](https://github.com/github/site-policy/blob/master/Policies/github-community-guidelines.md)
* [github-privacy-statement.md](https://github.com/github/site-policy/blob/master/Policies/github-privacy-statement.md)
* [github-terms-of-service.md](https://github.com/github/site-policy/blob/master/Policies/github-terms-of-service.md) (that's the longest one)
* [github-trademark-policy.md](https://github.com/github/site-policy/blob/master/Policies/github-trademark-policy.md) (just to be on the safe side)
* [global-privacy-practices.md](https://github.com/github/site-policy/blob/master/Policies/global-privacy-practices.md): that's the part relevant to us in EU (privacy shield etc)

What I didn't mention explicitly is in the sections between "good to know" and "interesting to read" – not especially relevant *to the project* as such, but to everyone using Github.

**Comment by danielegobbetti** [comment]
Thanks for identifying what's not relevant for us! It's good to have a shorter to do list :-)

**Comment by IzzySoft** [comment]
That was exactly the intention: helping you focus on the relevant stuff. Was of course not saying you're not allowed to read the rest :grin:

**Comment by IzzySoft** [comment]
Guess that's outdated now – and soon becomes obsolete anyhow, once our migration to Codeberg is complete. Closing it hence.
